packages/amplify-cli/src/extensions/amplify-helpers/update-amplify-meta.ts

```diff
@@ -64,7 +64,7 @@ function moveBackendResourcesToCurrentCloudBackend(resources: $TSObject[]) {
     // in the case that the resource is being deleted, the sourceDir won't exist
     if (fs.pathExistsSync(sourceDir)) {
       fs.copySync(sourceDir, targetDir);
-      if (resource?.service === ServiceName.LambdaFunction) {
+      if (resource?.service === ServiceName.LambdaFunction || resource?.service.includes('custom')) {
```

Comment:
should this be category or service?

Reply:
Please make sure push/pull works with this. Also is there hashing logic for node_modules when running amplify-status?

Reply:
``` 
 "custom": {
    "customResource9bcf0700": {
      "service": "customCDK",
      "providerPlugin": "awscloudformation",
      "providerMetadata": {
        "s3TemplateURL": "https://s3.amazonaws.com/amplify-nodemodulesremove-dev-195034-deployment/amplify-cfn-templates/custom/customResource9bcf0700-cloudformation-template.json",
        "logicalId": "customcustomResource9bcf0700"
      },
```